tests/driver_lis2dh12/main.c

```diff
@@ -35,6 +38,33 @@ static char str_out[3][8];
 /* allocate device descriptor */
 static lis2dh12_t dev;
 
+/* interrupt pins*/
+//TODO: change the interruptpins to real connected
```

Comment:
Definition of interrupt pins and enabling / disabling code sections should be done using macros.

Reply:
I added `LIS2DH12_INTERRUPT_TEST` to `main.c`. Maybe there is a better place...